catalog/app/utils/perspective.ts

```diff
@@ -18,7 +19,8 @@ export function renderViewer(
 ): HTMLPerspectiveViewerElement {
   const element = document.createElement('perspective-viewer')
   if (className) {
```

Comment:
seems like this check is not needed anymore, bc 'perspective-viewer-material' class should be always set

Reply:
Right, thanks